tests/scripts/test_file_content_bulk.sh

```diff
@@ -6,8 +6,6 @@
 pulp debug has-plugin --name "pulp_file" || exit 3
 
 cleanup() {
-  rm test_1.txt test_2.txt test_3.txt
-  rm add_content.json remove_content.json
```

Comment:
Let's keep those files around a bit for post mortem analysis.
pytest will clean them up eventually.